File: Java/Sumof2num.java
import java.util.Scanner;

public class Sumof2num{
    public static void main(String arg[]){
        int a,b;
        try (Scanner s=new Scanner(System.in)) {
            System.out.println("Enter the  two numbers:");
            a=s.nextInt();
            b=s.nextInt();
        }
        System.out.println("The Sum of two numbers is:"+(a+b));
    }
}

File: Python/Sumof2num.py
a=int(input("Enter the first number:"))
b=int(input("Enter the second number:"))
print("The sum of the two numbers:",a+b)
print('This works?')
c,d,e=[int(x) for x in input('Enter the three numbers now:').split()]
print('The sum of the two numbers is:',c+d+e)
x,y=(int(input()) for _ in range(2))
print(x+y)